CHANGELOG.md

```diff
@@ -1,4 +1,9 @@
 ## 1.10.0 (Unreleased)
```

Comment:
Please don't put the changelog entry in this PR (see [1] for more details). We (the maintainers of this provider) will make sure that changelog is correctly filled in before release.

[1] https://www.terraform.io/docs/extend/community/maintainers.html#versioning-and-changelog